smssync/src/main/java/org/addhen/smssync/presentation/service/SmsReceiverService.java

```diff
@@ -58,9 +58,6 @@
 public class SmsReceiverService extends Service implements HasComponent<AppServiceComponent> {
 
     @Inject
-    PostMessage mProcessMessage;
-
-    @Inject
     FileManager mFileManager;
```

Comment:
I think it still is - it just looks weird in the diff.  Take a look at the full file: https://github.com/alxndrsn/SMSSync/blob/401/smssync/src/main/java/org/addhen/smssync/presentation/service/SmsReceiverService.java#L60